Neuron/Sections/Dapp/WebView/DappViewController.swift

```diff
@@ -95,8 +97,13 @@ extension DappViewController: WKScriptMessageHandler {
     func userContentController(_ userContentController: WKUserContentController, didReceive message: WKScriptMessage) {
         switch message.name {
         case "pushSearchView":
-            let sCtrl = UIStoryboard(name: "DAppBrowser", bundle: nil).instantiateViewController(withIdentifier: "searchAppController")
-            self.navigationController?.pushViewController(sCtrl, animated: true)
+            let searchAppController = UIStoryboard(name: "DAppBrowser", bundle: nil).instantiateViewController(withIdentifier: "searchAppController")
```

Comment:
If these view controllers are from the same storyboard, just use `storyboard` instead of creating a new `UIStoryboard` instance.